src/discordRPC/version.json

```diff
@@ -1,6 +1,6 @@
 {
     "OBS:": "THIS IS A TEST VERSION ONLY FOR DELEVOPING PURPOSES",
-    "version": "0.1.0",
+    "version": "0.3.0",
```

Comment:
this is the test version, the original file is in the root of the project.

https://github.com/ThiaudioTT/hoi4-presence/blob/main/version.json